src/main/java/com/fasterxml/jackson/databind/cfg/JsonNodeFeature.java

```diff
@@ -79,7 +79,30 @@ public enum JsonNodeFeature implements DatatypeFeature
      *
      * @since 2.16
      */
-    FAIL_ON_NAN_TO_BIG_DECIMAL_COERCION(false)
+    FAIL_ON_NAN_TO_BIG_DECIMAL_COERCION(false),
+
+
+    /**
+     * Determines whether floating-point numbers should be deserialized into
+     * {@link java.math.BigDecimal} when reading {@link com.fasterxml.jackson.databind.JsonNode}s.
+     * This feature provides more precise control over number deserialization for {@code JsonNode}
+     * and takes precedence over {@link com.fasterxml.jackson.databind.DeserializationFeature#USE_BIG_DECIMAL_FOR_FLOATS}
+     * if explicitly set.
```

Comment:
if explicitly set to true

Reply:
I think we may want to discuss [this](https://github.com/FasterXML/jackson-databind/pull/5058/files#r2021314184) other comment first